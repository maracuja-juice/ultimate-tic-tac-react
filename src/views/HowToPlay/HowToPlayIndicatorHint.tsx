import * as React from 'react';
import styles from './HowToPlayDialog.module.css';
import { Tile } from '../../components/Board/Tile/Tile';
import { TileValue } from '../../state/AppState';

export function HowToPlayIndicatorHint() {
  return (
    <div className={styles.howToPlayIndicator}>
      <Tile
        value={TileValue.Empty}
        isTileRound={false}
        clickable={true}
        animate={false}
      />
    </div>
  );
}
