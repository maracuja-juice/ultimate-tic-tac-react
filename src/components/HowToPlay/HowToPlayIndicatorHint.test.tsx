import * as React from 'react';
import { shallow } from 'enzyme';
import { HowToPlayIndicatorHint } from './HowToPlayIndicatorHint';

describe('HowToPlayIndicatorHint', () => {
  it('should match snapshot', () => {
    const howToPlayIndicatorHint = shallow(<HowToPlayIndicatorHint />);
    expect(howToPlayIndicatorHint).toMatchSnapshot();
  });
});
