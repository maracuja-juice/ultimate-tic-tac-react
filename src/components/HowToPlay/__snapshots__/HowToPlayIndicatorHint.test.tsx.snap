// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowToPlayIndicatorHint should match snapshot 1`] = `
<div
  className="howToPlayIndicator"
>
  <Tile
    animate={false}
    clickable={true}
    isTileRound={false}
    value={2}
  />
</div>
`;
